src/js/components/breadcrumbs/JobsBreadcrumbs.js

```diff
@@ -3,104 +3,110 @@ import prettycron from "prettycron";
 import React from "react";
 import { Tooltip } from "reactjs-components";
 
+import Job from "#SRC/js/structs/Job";
+import JobTree from "#SRC/js/structs/JobTree";
+import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
+
 import Breadcrumb from "../../components/Breadcrumb";
 import BreadcrumbSupplementalContent
   from "../../components/BreadcrumbSupplementalContent";
 import BreadcrumbTextContent from "../../components/BreadcrumbTextContent";
 import Icon from "../Icon";
 import PageHeaderBreadcrumbs from "../../components/PageHeaderBreadcrumbs";
 
-function getJobStatus(jobStatus) {
-  if (jobStatus != null) {
-    return (
-      <BreadcrumbSupplementalContent>
-        <div className="service-page-header-status muted">
-          ({jobStatus})
-        </div>
-      </BreadcrumbSupplementalContent>
-    );
+function getItemStatus(item) {
+  if (!(item instanceof Job)) {
```

Comment:
both `getItemSchedule` and `getItemStatus` perform this check `!(item instanceof Job)` does it make sense to extract the check on the upper level?